requirements/development.txt

```diff
@@ -241,20 +244,15 @@ s3transfer==0.6.1
 sqlalchemy-bigquery==1.10.0
     # via apache-superset
 sqloxide==0.1.33
-    # via -r requirements/development.in
+    # via apache-superset
 stack-data==0.6.2
     # via ipython
 statsd==4.0.1
-    # via -r requirements/development.in
+    # via apache-superset
 tableschema==1.20.10
     # via apache-superset
-thrift==0.20.0
-    # via
-    #   apache-superset
-    #   pyhive
-    #   thrift-sasl
-thrift-sasl==0.4.3
```

Comment:
@mistercrunch the removal of `thrift-sasl` (which should be included via `pyhive[hive_pure_sasl]`) is breaking CI, i.e., for [test-postgres-hive](https://github.com/apache/superset/actions/runs/8532143694/job/23373369856?pr=27843#logs), I'm seeing:

```
> import thrift_sasl
E ModuleNotFoundError: No module named 'thrift_sasl'
```

Reply:
I think that was bad merge conflict resolution, but interestingly this PR brought it back - guessing through a `pip-compile-multi --no-upgrade` run -> https://github.com/apache/superset/pull/27631/files#diff-f449835486933e5cab23a8567940e46d53edd89923cf1a3a4fb83005b069a93fR260

Now I did another run here -> https://github.com/apache/superset/pull/27861/